src/smart-components/access-management/UsersTable.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import { useDataViewSelection, useDataViewPagination } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { BulkSelect, BulkSelectValue } from '@patternfly/react-component-groups/dist/dynamic/BulkSelect';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { Pagination } from '@patternfly/react-core';
+import { ActionsColumn } from '@patternfly/react-table';
+import { fetchUsers } from '../../helpers/user/user-helper';
+
+const COLUMNS = ['Username', 'Email', 'First name', 'Last name', 'Status', 'Org admin'];
+
+const ROW_ACTIONS = [
+  { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
+  { title: 'Remove from user group', onClick: () => console.log('REMOVE FROM USER GROUP') },
+];
+
+const PER_PAGE_OPTIONS = [
+  { title: '5', value: 5 },
+  { title: '10', value: 10 },
+  { title: '20', value: 20 },
+  { title: '50', value: 50 },
+  { title: '100', value: 100 },
+];
+
+const OUIA_ID = 'iam-users-table';
+
+const UsersTable = () => {
+  const [users, setUsers] = useState([]);
+  const [totalCount, setTotalCount] = useState(0);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const pagination = useDataViewPagination({ perPage: 20 });
+  const { page, perPage, onSetPage, onPerPageSelect } = pagination;
+
+  const selection = useDataViewSelection({ matchOption: (a, b) => a[0] === b[0] });
+  const { selected, onSelect, isSelected } = selection;
+
+  useEffect(() => {
+    const loadUsers = async () => {
+      setLoading(true);
+      try {
+        const result = await fetchUsers({
+          limit: perPage,
+          offset: (page - 1) * perPage,
+          orderBy: 'username',
+        });
+        setUsers(result.data);
+        setTotalCount(result.meta.count);
+      } catch (err) {
+        setError('Failed to load users');
+        console.error(err);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    loadUsers();
+  }, [page, perPage]);
```

Comment:
It would be better to use Redux instead of directly fetching the data here - we can use similar logic to the former users page. Also, the promise middleware will help us catch the errors more gracefully.